tests/settings.py

```diff
@@ -90,6 +90,14 @@
         "HOST": "",
         "PORT": "",
     },
+    "postgres_2": {
+        "ENGINE": "django.db.backends.postgresql_psycopg2",
+        "NAME": "admin_interface_2",
+        "USER": "postgres",
+        "PASSWORD": "postgres",
+        "HOST": "",
+        "PORT": "",
+    },
```

Comment:
I think there should be a replica also for the `sqlite` database (both databases are used in CI).

Reply:
sqlite uses `:memory:` in the config. So each db entry has a separate copy in-memory. That's the reason sqlite tests are passing right now.

Reply:
Maybe is the case to set it to a local file?